pkg/rpcmiddleware/rpclog/rpclog.go

```diff
@@ -57,14 +61,22 @@ func (l *ttnGrpcLogger) Infoln(args ...interface{}) {
 func (l *ttnGrpcLogger) Infof(format string, args ...interface{}) {
 	l.info(fmt.Sprintf(format, args...))
 }
+func (l *ttnGrpcLogger) warning(msg string) {
```

Comment:
Shouldn't we should keep this consistent with the `pkg/log` interface? Also `warning` is not a verb so it's not a good function name.

Reply:
Renamed to `warn`. How do you mean consistent here?

Reply:
Basically make the function `Warn(msg string)` as in our `Interface`, which you've already done.. :)